src/realm.js

```diff
@@ -1220,8 +1215,13 @@ export class Realm {
   incorporatePriorSavedCompletion(priorCompletion: void | PossiblyNormalCompletion) {
     if (priorCompletion === undefined) return;
     if (this.savedCompletion === undefined) {
+      // priorCompletion must be a previous savedCompletion, so the corresponding tracking maps would have been
```

Comment:
Any way to make this text into an invariant?

Reply:
The  closest proxy to it that can be checked cheaply is to require priorCompletion.savedEffects to be defined.